website/docs/plugin/framework/handling-data/types/index.mdx

```diff
@@ -60,3 +60,15 @@ This type is associated with:
 | Type | Use Case |
 |----------------|----------|
 | [Object](/terraform/plugin/framework/handling-data/types/object) | Mapping of explicit attribute names to values |
+
+### Tuple Type
+
+Type that defines an ordered collection of elements where each element has it's own type.
+
+<Note>
+This type intentionally includes less functionality than other types in the type system as it has limited real world application and therefore is not exposed to provider developers.
```

Comment:
I think we can go ahead and make this forward statement now so we don't have to update it later:

```suggestion
This type intentionally includes less functionality than other types in the type system as it has limited real world application and therefore is not exposed to provider developers except when working with dynamic values.
```